src/components/Image.ts

```diff
@@ -46,21 +54,39 @@ export class Image extends Component<ImageProps, ImageChild> {
 	public static readonly mixed: boolean = false;
 
 	#relationshipId: string | null = null;
+	#relationshipIdSvg: string | null = null;
 
 	#location = `word/media/${createRandomId('img')}`;
 	get location() {
 		return this.#location;
 	}
+	#locationSvg = `word/media/${createRandomId('svg')}`;
+	get locationSvg() {
```

Comment:
What's the `locationSvg` getter used for?

Reply:
it is used in `ensureRelationship`, same as `#location`. In fact, I, at first, added it for consistency with `#location` but is is used now